dataflowAPI/src/stackanalysis.C

```diff
@@ -588,7 +588,7 @@ void StackAnalysis::summarize() {
       Block *block = bIter->first;
       for (auto aIter = (*intervals_)[block].begin();
          aIter != (*intervals_)[block].end(); aIter++) {
-         Address addr = aIter->first;
+         //Address addr = aIter->first;
```

Comment:
Any reason to not just delete these?

Reply:
left these for documentation of the unused first and second members